cmd/timescale-prometheus/main.go

```diff
@@ -162,6 +165,14 @@ func main() {
 	log.Info("msg", "Version:"+version.Version+"; Commit Hash: "+version.CommitHash)
 	log.Info("config", util.MaskPassword(fmt.Sprintf("%+v", cfg)))
 
+	err = waitForDb(cfg)
+	if err != nil {
```

Comment:
Since `waitForDb` only returns an error, I think it will be better to have them on same line. 
Much like,
```
if err = waitForDb(cfg); err != nil {
  ...
}
```

Reply:
So I actually disagree with that one; I think the hit to readability from the singleline version outweighs the pollution reduction of not having to define `err` outside.

Reply:
(incidentally, I'd consider the [go style guide's](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow) attitude towards error handling a tacit admission that the single-line form is usually bad style)